### ISSUE TITLE
Middle clicking to select a block while shifting = InventoryMove false positive

### ISSUE THREAD

**Issue by Lysandr0** [CaptainObvious0/NoCheatPlus#24]
### Full output of /ncp version
```

 [INFO] ---- Version information ----
 [INFO] #### Server ####
 [INFO] git-Spigot-e6eb36f-d3ed151 ~MC: 1.13.2~
[INFO] detected: 1.13.2
 [INFO] #### NoCheatPlus ####
 [INFO] Plugin: 3.16.1-SNAPSHOT-CustomFork-v1.3-beta.3
 [INFO] MCAccess: 1.4.5-1.12.2|? / CB-Reflect
 [INFO] Features:
[INFO] blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8 | BlocksMC1_9 | BlocksMC1_10 | BlocksMC1_11 | BlocksMC1_12 | BlocksMC1_13 [INFO] checks: FastConsume | Gutenberg | HotFixFallingBlockPortalEnter | AttackFrequency | FlyingFrequency | KeepAliveFrequency
[INFO] defaults: pvpKnockBackVelocity
 [INFO] packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance | DebugAdapter
 [INFO] Hooks: AllViolations~NCP~ 1.0
[INFO] #### Related Plugins ####
[INFO] ProtocolLib v4.4.0 | ProtocolSupport v4.29-dev

```
### Describe the issue
As the title says :p
it happens with the other conditions as well (Swimming, sprinting etc...), the hotbar should probably be exempted from being checked.

### Extra links/Videos (Including debug logs)
https://www.dropbox.com/s/ahtu3bjg1z1vy7a/InventoryMove_FP.mov?dl=0

**Comment by CaptainObvious0** [comment]
Thanks for reporting. I’ll fix this issue as well as add an ignore creative value in the config.

**Comment by CaptainObvious0** [comment]
Fixed with 3d9d66e
